src/Endpoint/SearchEndpoint.php

```diff
@@ -101,7 +101,7 @@ public function setSort(Sort $sort): self
     /**
      * @inheritDoc
      */
-    public function getSort(): Sort
+    public function getSort(): array
```

Comment:
As I understand it can be empty.
```suggestion
    public function getSort(): ?array
```